ApplyToBecomeInternal/ApplyToBecomeInternal/Pages/Components/GeneralInformation/default.cshtml

```diff
@@ -1,21 +1,13 @@
 ﻿@model GeneralInformationViewModel
 @{
-	var kw = WidthOverride.TwoThirds;
-	var distance = "";
+		var kw = WidthOverride.TwoThirds;
+		var distance = "";
 
-	if (!string.IsNullOrEmpty(Model.DistanceFromSchoolToTrustHeadquarters))
-	{
-		distance = $"{Model.DistanceFromSchoolToTrustHeadquarters} miles";
-		if (!string.IsNullOrEmpty(Model.DistanceFromSchoolToTrustHeadquartersAdditionalInformation))
```

Comment:
Please double check the indentation for the if{} statement code block.

Reply:
checked  - no extra tab present